docker/run.sh

```diff
@@ -42,4 +42,4 @@ if [ -z "$SYSFS_CGROUP_FILES" ]; then
 	fi;
 fi;
 
-exec /play/play run -XshowSettings:vm -XX:+UseSerialGC -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
+exec /play/play run -XshowSettings:vm -XX:+UseSerialGC -XX:+UnlockExperimentalVMOptions
```

Comment:
See my comment in the original issue!

Just keep `-XshowSettings:vm`, but please verify that the Docker container still accounts cgroup limits when you bind mount them.
See commit 9fdf38ed8db95527eb43c3a31a0684be14472643.

Reply:
While at it, please rephrase the commit message too! Thanks

E.g. "Docker: fix broken and xxxxx (legacy?) JVM options" and fill the details, like I did in commit ff77348a2adaf1eea63bccb292240592ce49e8ea!

Reply:
Couldn't amend the last commit as I did it via Github Web and there is no such option yet 🤔

Reply:
OK, I'll squash it for you!

I'll rebase all the PR on my branch anyway! :)

Reply:
Thanks :)